scripts/docsgen/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/coder/envbuilder"
+)
+
+func main() {
+	readmePath := "README.md"
+	readmeFile, err := os.ReadFile(readmePath)
+	if err != nil {
+		panic("Error reading " + readmePath + " file")
+	}
+	readmeContent := string(readmeFile)
+	startIndex := strings.Index(readmeContent, "<!--- start-flags --->")
```

Comment:
Perhaps these could be defined at the top of the file as:

```
const (
	startSection = "<!-- ..."
	endSection = "<!-- ..."
)
```

Easier maintenance if changing it in the future.